app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
```

Comment:
Place each argument on a new line including `self`, so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability.